commands/invite.js

```diff
@@ -3,8 +3,8 @@ module.exports.execute = async (client, message, args) => {
 	let inviteText = 'Invite to the KOA server: https://discord.gg/EYX7XGG';
 
 	if(args[0]) {
-		if(args[0].toLowerCase() === 'camelot') {
-			inviteText = 'Invite to the Knights of Academia: Camelot server: https://discord.gg/wu3a6JA';
+		if(args[0].toLowerCase() === 'jesters') {
```

Comment:
I think we should add a case for camelot still, while people are getting used to it, so we have:

camelot/jesters/joa

Reply:
Will add this in tonight for invite, just committed your suggestion on the info.